src/main/java/org/reaktivity/command/log/internal/LogCommand.java

```diff
@@ -54,27 +55,41 @@ public static void main(String[] args) throws Exception
             String directory = cmdline.getOptionValue("directory");
             boolean verbose = cmdline.hasOption("verbose");
             String type = cmdline.getOptionValue("type", "streams");
+            int interval = Integer.parseInt(cmdline.getOptionValue("interval", "0"));
 
             Properties properties = new Properties();
             properties.setProperty(Configuration.DIRECTORY_PROPERTY_NAME, directory);
 
             final Configuration config = new LogCommandConfiguration(properties);
 
+            Command command = null;
```

Comment:
Can we make this a list of commands that we iterate, so we can run multiple of them in intervals?

`-t queues -t counters -i 10` runs both a queues and counters commands